raft/tracker/inflights.go

```diff
@@ -56,11 +56,9 @@ func (in *Inflights) Add(inflight uint64) {
 	if in.Full() {
 		panic("cannot add into a Full inflights")
 	}
-	next := in.start + in.count
-	size := in.size
-	if next >= size {
-		next -= size
-	}
+
+	next := (in.start + in.count) % in.size
```

Comment:
If `in.start + in.count` is more than twice as large as `in.size`, `next` will be changed compared to before. And the `next` will decide whether to execute `grow`. Maybe some context I don't know.

Reply:
Thank you for your kind reply.